test/provisioner/provision_ibmcloud.go

```diff
@@ -688,6 +689,8 @@ func (p *IBMCloudProvisioner) DeleteCluster(ctx context.Context, cfg *envconf.Co
 			continue
 		}
 		if foundClr == nil {
+			// Waiting sometime to let cluster releated vsis be deleted
+			time.Sleep(120 * time.Second)
```

Comment:
Didn't find any `.network_interfaces` in the output json, but it seems that we can check the `reserved_ips` section.
I update the codes.

Reply:
Thanks - as I've been hitting the stop-ship bug in my testing the peer pods doesn't get cleaned up for me, so that might be why I have network interfaces, but if there is a more reliable way that's great.

Reply:
Add the attached loadBalancers for a subnet, to make sure we can remove the subnet successfully.

Reply:
```
...
time="2023-04-13T01:45:43-07:00" level=info msg="Install CoCo and cloud-api-adaptor"
Wait for the cc-operator-daemon-install DaemonSet be available
Wait for the pod cc-operator-daemon-install-m8698 be ready
Wait for the cloud-api-adaptor-daemonset DaemonSet be available
Wait for the pod cloud-api-adaptor-daemonset-ml46w be ready
Wait for the kata runtimeclass be created
=== RUN   TestCreateSimplePod
=== RUN   TestCreateSimplePod/Simple_Peer_Pod
=== RUN   TestCreateSimplePod/Simple_Peer_Pod/PodVM_is_created
time="2023-04-13T01:48:43-07:00" level=info msg="PodVM name: simple-peer-pod"
time="2023-04-13T01:48:44-07:00" level=debug msg="Instance number: 0, Instance id: 0797_eb758a5f-c799-4448-a960-ad81473ec273, Instance name: podvm-simple-peer-pod-6c055667"
--- PASS: TestCreateSimplePod (76.59s)
    --- PASS: TestCreateSimplePod/Simple_Peer_Pod (76.59s)
        --- PASS: TestCreateSimplePod/Simple_Peer_Pod/PodVM_is_created (1.05s)
PASS
time="2023-04-13T01:48:45-07:00" level=info msg="Deleting Cluster liudali-e2e-test-xamd64."
time="2023-04-13T01:48:46-07:00" level=info msg="Waiting for cluster liudali-e2e-test-xamd64 to be removed..."
time="2023-04-13T01:48:46-07:00" level=info msg="Waited 0 minutes..."
time="2023-04-13T01:49:47-07:00" level=info msg="Waited 1 minutes..."
time="2023-04-13T01:50:48-07:00" level=info msg="Waited 2 minutes..."
time="2023-04-13T01:51:48-07:00" level=info msg="Waited 3 minutes..."
time="2023-04-13T01:54:49-07:00" level=info msg="Cluster liudali-e2e-test-xamd64 is removed."
time="2023-04-13T01:54:52-07:00" level=info msg="SSH Key daily-e2e-test-vsi-key with ID r018-8b777344-16f4-4bd8-8974-c242122a4fad is deleted."
time="2023-04-13T01:54:52-07:00" level=info msg="Found subnet liudali-e2e-test-xamd64-vpc-subnet with ID 0797-a2355059-8a7f-4da2-b537-d0a4adc2adce."
time="2023-04-13T01:54:54-07:00" level=info msg="Public Gateway r018-81664662-575f-41df-a432-8eaef4377d0a was unattached from Subnet 0797-a2355059-8a7f-4da2-b537-d0a4adc2adce."
time="2023-04-13T01:54:54-07:00" level=info msg="Public Gateway r018-81664662-575f-41df-a432-8eaef4377d0a was deleted."
time="2023-04-13T01:54:54-07:00" level=info msg="Waiting for attached LoadBalancers to be removed from liudali-e2e-test-xamd64-vpc-subnet ..."
time="2023-04-13T01:54:57-07:00" level=info msg="All attached LoadBalancers are removed from liudali-e2e-test-xamd64-vpc-subnet ..."
time="2023-04-13T01:54:57-07:00" level=info msg="Deleting subnet with ID 0797-a2355059-8a7f-4da2-b537-d0a4adc2adce."
time="2023-04-13T01:54:58-07:00" level=info msg="Deleted subnet with ID 0797-a2355059-8a7f-4da2-b537-d0a4adc2adce."
time="2023-04-13T01:54:59-07:00" level=info msg="Found VPC liudali-e2e-test-xamd64-vpc with ID r018-7807d7b6-7ed4-442a-95f5-2869c869f899."
time="2023-04-13T01:54:59-07:00" level=info msg="Deleting VPC with ID r018-7807d7b6-7ed4-442a-95f5-2869c869f899."
time="2023-04-13T01:54:59-07:00" level=info msg="Deleted VPC with ID r018-7807d7b6-7ed4-442a-95f5-2869c869f899."
ok  	github.com/confidential-containers/cloud-api-adaptor/test/e2e	1851.038s
root@peer-pod1 cloud-api-adaptor (e2e) $
```